instrumentation/opentelemetry-instrumentation-aiohttp-client/src/opentelemetry/instrumentation/aiohttp_client/__init__.py

```diff
@@ -108,7 +108,7 @@ def response_hook(span: Span, params: typing.Union[
 from opentelemetry.trace.status import Status, StatusCode
 from opentelemetry.util.http import remove_url_credentials
 
-_UrlFilterT = typing.Optional[typing.Callable[[str], str]]
+_UrlFilterT = typing.Optional[typing.Callable[[yarl.URL], str]]
```

Comment:
Did any older version use `str`? In that case probably better to have a union type.

Reply:
No, it's `yarl.URL` more than 4 years
https://github.com/aio-libs/aiohttp/blame/0e79469381418107e81670dc4e299ad56fdc5354/aiohttp/tracing.py#L214